.gitlab-ci.yml

```diff
@@ -4,39 +4,30 @@ stages:
   - trigger_prep
   - trigger
 
-variables:
-  GOLANG_VERSION: "1.17.13"
-
 include:
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-lint.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-commits.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-license.yml'
 
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+
 test:
   stage: test
-  image: golang:1.17
+  image: debian:11
```

Comment:
My experience is that Fedora provides the newest versions of compilers, linters, static checkers,... which often mean more issue found. But not a super-strong preference here. For inspiration, this is what we have in CFEngine/Core: https://github.com/cfengine/core/blob/master/Makefile.am#L170

Reply:
It's more about convention on the Mender team. We generally stick to alpine if we can, mainly due to speed and size benefits, and then move to either Ubuntu or Debian if we need something a bit more esoteric. So far it hasn't let us down, so I think introducing a third package manager is unnecessary, at least for now.

Reply:
Yeah, that's what I was afraid of. :wink: Just to support my point:
```
$ gcc --version
gcc (Debian 10.2.1-6) 10.2.1 20210110
$ clang --version
Debian clang version 11.0.1-2
```
on Debian 11,  vs. this on Fedora 36:
```
$ gcc --version
gcc (GCC) 12.2.1 20221121 (Red Hat 12.2.1-4)
$ clang --version
clang version 14.0.5 (Fedora 14.0.5-1.fc36)
```
and https://github.com/cfengine/core/pull/4959#issue-1249629787